docs/config.md

```diff
@@ -0,0 +1,266 @@
+
+**Versions: 2.x**
+
+:::success Client-side configuration
+
+Using Redoc as a standalone (HTML or React) tool, these options must be presented in [kebab case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case). 
+For example, `scrollYOffset` becomes `scroll-y-offset`, and `expandResponses` becomes `expand-responses`.
+
+:::
+
+## Functional settings
+
+
+### disableSearch
+
+Disables search indexing and hides the search box from the API documentation page.
+
+### minCharacterLengthToInitSearch
+
+Sets the minimum amount of characters that need to be typed into the search dialog to initiate the search.
+
+_Default: 3_
+
+### expandDefaultServerVariables
+
+Enables or disables expanding default server variables.
+
+### expandResponses
+
+Controls which responses to expand by default. Specify one or more responses by providing their response codes as a comma-separated list without spaces, e.g. `expandResponses='200,201'`. Special value 'all' expands all responses by default. Be careful: this option can slow down documentation rendering time.
+
+### expandSingleSchemaField
+
+Automatically expands the single field in a schema.
+
+### hideDownloadButton
+
+Hides the 'Download' button for saving the API definition source file. **This does not make the API definition private**, it just hides the button.
+
+### hideHostname
+
+If set to `true`, the protocol and hostname are not shown in the operation definition.
+
+### hideLoading
+
+Hides the loading animation. Does not apply to CLI or Workflows-rendered docs.
+
+### hideRequestPayloadSample
+
+Hides request payload examples.
+
+### hideOneOfDescription
+
+If set to `true`, the description for `oneOf`/`anyOf` object is not shown in the schema.
+
+### hideSchemaPattern
+
+If set to `true`, the pattern is not shown in the schema.
+
+### hideSchemaTitles
+
+Hides the schema title next to to the type.
+
+### hideSecuritySection
+
+Hides the Security panel section.
+
+### hideSingleRequestSampleTab
+
+Hides the request sample tab for requests with only one sample.
+
+### htmlTemplate
+
+Sets the path to the optional HTML file used to modify the layout of the reference docs page.
+
+### jsonSampleExpandLevel
+
+Sets the default expand level for JSON payload samples (response and request body). The default value is 2, and the maximum supported value is '+Infinity'. It can also be configured as a string with the special value `all` that expands all levels.
+
+_Default: 2_
+
+### maxDisplayedEnumValues
+
+Displays only the specified number of enum values. The remaining values are hidden in an expandable area. If not set, all values are displayed.
+
+### menuToggle
+
+If set to `true`, selecting an expanded item in the sidebar twice collapses it.
+
+_Default: true_
+
+### nativeScrollbars
+
+If set to `true`, the sidebar uses the native scrollbar instead of perfect-scroll. This is a scrolling performance optimization for big API definitions.
```

Comment:
```suggestion
If set to `true`, the sidebar uses the native scrollbar instead of perfect-scroll. This setting is a scrolling performance optimization for big API definitions.
```